samples/ordered_puzzle/components/ordered_puzzle.js

```diff
@@ -35,18 +36,29 @@ AFRAME.registerComponent("ordered-puzzle", {
       this.puzzlePiece.setAttribute("height", data.images[i].height);
       this.puzzlePiece.setAttribute(
         "position",
-        data.images[i].x + " " + data.images[i].y + " 0"
+        (Math.random() - 0.5) *
+          (blackboard.getAttribute("geometry").width - data.images[i].width) +
+          " " +
+          (Math.random() - 0.5) *
+            (blackboard.getAttribute("geometry").height -
+              data.images[i].height) +
+          " 0"
```

Comment:
Just for my understanding, this is to randomize starting position of the sub-images? Can we add a comment describing this

Reply:
Yep exactly, I'll add a comment as well